### PR TITLE
Login Page

### PR DESCRIPTION
**Pull request by saadali5997**

### CODE REVIEW

File: templates/registration/login-head.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{% load static %}
+
+<!-- begin::Head -->
+
+<head>
```
Comment (Ayub-Khan) [inline templates/registration/login-head.html:5]: can we use the same head we are using in base ?

File: templates/registration/login-scripts.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{% load static %}
+<!-- begin::Global Config(global config for global JS sciprts) -->
+<script>
+    var KTAppOptions = {
```
Comment (Ayub-Khan) [inline templates/registration/login-scripts.html:4]: Do we need those scripts ?

**Review by Ayub-Khan — APPROVED** [review]

File: templates/registration/login-head.html (hunk 1 above)
Comment (saadali5997) [reply templates/registration/login-head.html:5]: @Ayub-Khan sir both heads have some different references as you can see [here](https://www.diffchecker.com/IvVlH5Kk).

File: templates/registration/login-scripts.html (hunk 2 above)
Comment (saadali5997) [reply templates/registration/login-scripts.html:4]: If we don't use those scripts, `Sign Up` and `Forgot Password` buttons will not work. Should I remove their block from login page?

Comment (Ayub-Khan) [reply templates/registration/login-scripts.html:4]: Lets keep those for the time being.

**Pull request merged**